lib/request_handler/fieldsets_parser.rb

```diff
@@ -29,13 +29,18 @@ def run
     private
 
     def parse_options(type, values)
+      return [] if allowed[type] == false
```

Comment:
see comment in the specs. I'd like to have this fail with an ExternalArgumentError stating that fieldset is not supported for this type.